### ISSUE TITLE
project configuration undocumented

### ISSUE THREAD

**Issue by hramrach** [openSUSE/obs-docu#337]
There is mention of project metadata and project configuration in

22 OBS Concepts > 22.1 Project Organization
https://openbuildservice.org/help/manuals/obs-user-guide/cha.obs.concepts#id-1.5.10.3.4

However, this is too deep in chapter hierachy and not shown in the index.

Also these are mentioned, not described.

**Comment by JanZerebecki** [comment]
Yes, the index does not show the 3rd and 4th level, but it should. I also get bitten by that and often can not find the right chapter. (The glossary also has no links to further explanations.)

Most project metadata are described in https://openbuildservice.org/help/manuals/obs-user-guide/cha-obs-basicworkflow but especially a link to it for explaining repositories is missing from the "Project Metadata" chapter on the concepts page. (But see https://github.com/openSUSE/obs-docu/issues/159 for things that are missing there.)
